input/data/2014/2014ApJ...785L..16A/tev-000067-1.yaml

```diff
@@ -0,0 +1,16 @@
+source_id: 67
+reference_id: 2014ApJ...785L..16A
+file_id: 1
+telescope: veritas
+
+data:
+  livetime: 28.5
+  significance: 8.5
+
+spec:
+  mjd: {min: 54881.0, max: 55003.0}
+  erange: {min: 0.12, max: 0.75, unit: TeV}
+  model:
+    type: pl
```

Comment:
This is all now on github (sorry, the flaky internet on the ICRC interrupted me sometimes exactly at the wrong time)